gradle.properties

```diff
@@ -7,6 +7,10 @@ dependency.analysis.print.build.health=true
 dependency.analysis.warnings.silent=true
 
 
+SONATYPE_HOST=DEFAULT
+RELEASE_SIGNING_ENABLED=true
```

Comment:
Prepares for the behavior change mentioned here https://github.com/vanniktech/gradle-maven-publish-plugin/blob/master/CHANGELOG.md#version-0200-2022-06-02